.github/workflows/vast.yaml

```diff
@@ -307,6 +307,7 @@ jobs:
           cmake -B "$BUILD_DIR" \
             -DCMAKE_INSTALL_PREFIX:STRING="${PWD}/opt/vast" \
             -DCPACK_PACKAGE_FILE_NAME:STRING="$PACKAGE_NAME" \
+            -DCPACK_PACKAGING_INSTALL_PREFIX:STRING="" \
```

Comment:
This triggers a warning in CI, so I think we should add a comment on why we're adding this.